tests/src/lv_test_conf.h

```diff
@@ -82,6 +82,9 @@ typedef void * lv_user_data_t;
 /* Simulate VG-Lite hardware using ThorVG */
 #define LV_USE_VG_LITE_THORVG       1
 
+/* VG-Lite GPU buffer alignment. */
+#define LV_DRAW_BUF_ALIGN           64
```

Comment:
It should set only in case of the VG-Lite test else `LV_DRAW_BUF_ALIGN       852` never will be applied.

Reply:
It does take effect only after testing under VG_LITE. It is judged above that `LV_TEST_OPTION == 6`.

Reply:
My bad, thanks!